.github/workflows/tests-ordinary-integration.yml

```diff
@@ -13,16 +13,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        file:
-          - test_itde_manager.py
-          - test_sagemaker_extension_wrapper.py
-          - itest_slc.py
-          - test_transformers_extension_wrapper.py
-          - itde_mgr_in_container/test_itde_connect.py
-          - itde_mgr_in_container/test_itde_external.py
-          - itde_mgr_in_container/test_itde_recreation_after_take_down.py
-          - itde_mgr_in_container/test_itde_recreation_without_take_down.py
-          - itde_mgr_in_container/test_itde_stop_and_restart.py
+        include:
+          - file: test_itde_manager.py
+            extra_param: ""
+          - file: test_sagemaker_extension_wrapper.py
+            extra_param: ""
+          - file: test_transformers_extension_wrapper.py
+            extra_param: ""
+          - file: itde_mgr_in_container/test_itde_connect.py
+            extra_param: ""
+          - file: itde_mgr_in_container/test_itde_external.py
+            extra_param: ""
+          - file: itde_mgr_in_container/test_itde_recreation_after_take_down.py
+            extra_param: ""
+          - file: itde_mgr_in_container/test_itde_recreation_without_take_down.py
+            extra_param: ""
+          - file: itde_mgr_in_container/test_itde_stop_and_restart.py
+            extra_param: ""
+          - file: itest_slc.py
+            extra_param: "--package-manager pip"
+          - file: itest_slc.py
+            extra_param: "--package-manager conda"
```

Comment:
Probably the `extra_param` needs to be evaluated here
```
poetry run -- pytest -rA --setup-show test/integration/ordinary/${{ matrix.file }}
```

Reply:
Done